### ISSUE TITLE
Inf log-probs when using score.py

### ISSUE THREAD

**Issue by zhangdongxu** [lisa-groundhog/GroundHog#45]
Meet this issue when implement score.py. And all the log-probs are inf. 
I test this on the training data, where in the training process, most target sentences can be recovered.
In my opinion, zero probability should not exist since in the softmax, there is inner product between two vector.
Any suggestions?

**Comment by nouiz** [comment]
Just to tell you that this repo isn't maintained anymore. See  the warning
on this page:

https://github.com/lisa-groundhog/GroundHog

I'm not sure someone else will reply. There is another project that you can
use instead of groundhog that is based on Theano.

**Comment by zhangdongxu** [comment]
Thank you. I notice that. But I concerned that has attention-based structure been employed in Block's translation system?

**Comment by rizar** [comment]
Yes, the implementation in Blocks uses attention.
